docs/multi-homing.md

```diff
@@ -22,8 +22,12 @@ configurations are defined in a CRD named `NetworkAttachmentDefinition`.
 Below you will find example attachment configurations for each of the current
 topologies OVN-K allows for secondary networks.
 
-**NOTE**: currently, all the secondary networks **only** allow for east/west
-traffic.
+**NOTE**:
+- currently, all the secondary networks **only** allow for east/west traffic.
+- networks are **not** namespaced - i.e. creating multiple
+  `network-attachment-definition`s with different configurations pointing at the
+  same network (same `NetConf.Name` attribute) with different configurations in
+  different namespaces is **not** supported.
```

Comment:
`with different configurations` is said twice.
Also wanted to avoid `in different namespaces` because is not really relevant, in is not allowed in different namespaces or in same namespace.

Reply:
Done.